package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "portable-mlir-toolchain",
+  "version": "1.0.0",
+  "description": "Portable LLVM/MLIR Toolchain",
+  "main": "./dist/index.js",
+  "scripts": {
+    "build": "npx ncc build src/index.ts -o dist --source-map --license licenses.txt",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/burgholzer/portable-mlir-toolchain.git"
+  },
+  "keywords": [],
+  "author": "burgholzer",
+  "license": "ISC",
```

Comment:
We definitely need to update this before finalising.
A couple other things in here need to be adjusted too once we move the repository elsewhere.